lib/disposable/twin/coercion.rb

```diff
@@ -18,7 +18,10 @@ def property(name, options={}, &block)
     end
 
     def coercing_setter!(name, type, nilify=false)
-     type = type ? (DRY_TYPES_CONSTANT::Nil | type) : DRY_TYPES_CONSTANT::Nil if nilify
+      # TODO: remove nilily with next release (0.5) for new dry-type versions
+      type = type ? (DRY_TYPES_CONSTANT::Nil | type) : DRY_TYPES_CONSTANT::Nil if nilify
+
+      warn "DEPRECATION WARNING [Disposable]: nilify is deprecated and it will be removed with the next release" if nilify && DRY_TYPES_CONSTANT == Types::Params
```

Comment:
I'm not sure that this is right. The Dry changes happened with the release of 1.x, but this is checking that the the dry is less than 0.13.0. 0.13.0 had the changes from Types::Form to Types::Params, but not the Nil changes.

Reply:
Yeah ok makes sense - going to change the check to 1.x